docroot/sites/its.uiowa.edu/modules/its_core/src/Plugin/Block/AlertTypeLegend.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Drupal\its_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides an 'Alert Type Legend' block.
+ *
+ * @Block(
+ *   id = "alert_type_legend_block",
+ *   admin_label = @Translation("Alert type legend"),
+ *   category = @Translation("Site custom")
+ * )
+ */
+class AlertTypeLegend extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return ['label_display' => FALSE];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $tids = ['406', '411', '416', '421'];
```

Comment:
can we make use of its_core_alert_type_tag_map() instead?

Reply:
tweaked a bit to expand the functionality of the helper function and fix the dependency injection issue with the block.